pkg/minikube/out/register/json.go

```diff
@@ -69,3 +71,9 @@ func PrintWarning(warning string) {
 	w := NewWarning(warning)
 	printAndRecordCloudEvent(w, w.data)
 }
+
+// RecordAudit records an Audit type in JSON format
+func RecordAudit(command string, args string, user string, startTime time.Time, endTime time.Time) {
+	a := NewAudit(command, args, user, startTime, endTime)
+	recordCloudEvent(a, a.data)
```

Comment:
how about making RecordCloudEvent public and then moving RecordAudit to audit package.
so that way all the audit related thigns will be in the audit package
and this package is only scoped down to doing one thing, "registering cloud event"

Reply:
Good idea, done!